deploy/operator/libvirt_disks.sh

```diff
@@ -0,0 +1,45 @@
+if [ -z "${NODES}" ]; then
+    export NODES=$(virsh list --name | grep worker || virsh list --name | grep master)
+fi
+
+if [ -z "${DISKS}" ]; then
+    echo "You must provide DISKS env-var. For example:"
+    echo "    DISKS=\$(echo sd{b..f}) bash ./libvirt_disks.sh create"
+    exit 1
+fi
+
+function create() {
+    export SIZE=${SIZE:-50G}
+
+    echo "Creating libvirt disks and attaching them..."
+    for node in ${NODES}; do
+        for disk in ${DISKS}; do
+            qemu-img create -f raw "/tmp/${node}-${disk}.img" "${SIZE}"
+            virsh attach-disk "${node}" "/tmp/${node}-${disk}.img" "${disk}"
+        done
+    done
+
+    echo "Done creating libvirt disks!"
+}
+
+function destroy() {
+    echo "Unattaching libvirt disks and destroying them..."
+
+    for node in ${NODES}; do
+        for disk in ${DISKS}; do
+            img_path="/tmp/${node}-${disk}.img"
+            if [ -f ${img_path} ]; then
+                virsh detach-disk "${node}" "${disk}"
+                rm -rf ${img_path}
+            fi
+        done
+    done
+
+    echo "Done destroying libvirt disks!"
+}
+
+case "$@" in
+    create)  create;;
+    destroy) destroy;;
+    *)       echo "Usage: DISKS=<disk names> bash ./libvirt_disks.sh (create|destroy)" && exit 1;;
```

Comment:
```suggestion
    *)       echo "Usage: DISKS=<disk names> bash ./libvirt_disks.sh ($(compgen -A function | paste -s -d'|')" && exit 1;;
```
will result all the available functions
```bash
Usage: DISKS=<disk names> bash ./libvirt_disks.sh (create|destroy)
```